gradle/java.gradle

```diff
@@ -23,7 +23,6 @@ allprojects {
 }
 
 subprojects {
-  apply plugin: 'java'
```

Comment:
Why is this removed?

Reply:
Duplicate of line 19, where it is applied via `allprojects{}`, which includes the root project.